server/routes/activitiesMigration/activitiesMigrationController.ts

```diff
@@ -94,12 +94,21 @@ export default class ActivitiesMigrationController {
         errors.push({ text: `Failed to check incentive levels due to error: ${error.data.userMessage}`, href: '' })
         return []
       }),
-    ]).then(([estimatedCount, dlqCount, incentiveLevels]) => {
+
+      this.nomisPrisonerService.checkServiceAgencySwitch(prisonId, 'ACTIVITY', context(res)).catch(error => {
+        errors.push({
+          text: `Failed to check if ACTIVITY feature switch turned on for ${prisonId}: ${error.data.userMessage}`,
+          href: '',
+        })
+        return 'Error checking ACTIVITY feature switch'
+      }),
+    ]).then(([estimatedCount, dlqCount, incentiveLevels, featureSwitchOn]) => {
       req.session.startActivitiesMigrationForm.estimatedCount = estimatedCount.toLocaleString()
       req.session.startActivitiesMigrationForm.dlqCount = dlqCount.toLocaleString()
       req.session.startActivitiesMigrationForm.incentiveLevelIds = incentiveLevels.map(
         (level: IncentiveLevel) => level.code,
       )
+      req.session.startActivitiesMigrationForm.prisonSwitchedOn = featureSwitchOn
```

Comment:
This is the main change - check that the prison has been feature switched on during the migration preview.